content/docs/reference/downstream-mtls-settings.mdx

```diff
@@ -172,7 +172,7 @@ The Ingress Controller does not support these settings.
 
    This mode also adds a requirement that any configured [**CRL**](#crl) is valid only between its `thisUpdate` and `nextUpdate` timestamps. _If an invalid CRL is provided for a certain CA, no certificates issued by that CA will be allowed._
 
-   Also note that in this mode, requests without a trusted client certificate will never reach the Pomerium authorize service, and so no record of these requests will be logged.
+   Selecting this mode will also enable TLS connection failure logging. Any connection attempt that fails during the TLS handshake will be logged by the proxy service with the message `"listener connection failure"`. The log entry will include the client IP address, server hostname, and client certificate info (if a client certificate was presented). This allows you to monitor rejected connections. (Otherwise, when a connection is rejected due to mTLS requirements, the client would never have the opportunity to make any requests, and so there would be no entries in the [access log](/docs/reference/access-log-fields#access-log-behavior) or [authorize log](/docs/capabilities/audit-logs).)
```

Comment:
@kenjenkins should we link to the authorize log reference setting page at https://www.pomerium.com/docs/reference/authorize-log-fields instead of the Audit Logs page?
```suggestion
   Selecting this mode will also enable TLS connection failure logging. Any connection attempt that fails during the TLS handshake will be logged by the proxy service with the message `"listener connection failure"`. The log entry will include the client IP address, server hostname, and client certificate info (if a client certificate was presented). This allows you to monitor rejected connections. (Otherwise, when a connection is rejected due to mTLS requirements, the client would never have the opportunity to make any requests, and so there would be no entries in the [access log](/docs/reference/access-log-fields#access-log-behavior) or [authorize log](/docs/reference/authorize-log-fields instead of the Audit Logs page?).)
```

Reply:
I was thinking the opposite actually — I was hoping we could link to a capabilities page for both of these, but I didn't see one for access logging, only the one for authorization logging.

Reply:
OK. That makes sense. Do you think we need a capabilities page for access logging? If so, happy to add this as a discussion item in next week's docs sync.

Reply:
I think this would make sense — or maybe this could be combined with the other information on the existing "Audit Logs" page. I can file an issue to track this idea.